collective/elasticsearch/hook.py

```diff
@@ -45,11 +45,14 @@ def index_batch(remove, index, positions, es=None):
                     '_id': uid
                 }
             })
-        es.connection.bulk(
+        result = es.connection.bulk(
             index=es.index_name,
             doc_type=es.doc_type,
             body=bulk_data)
 
+        if "errors" in result and result["errors"] is True:
```

Comment:
Is there any situation in which `errors` key isn't returned?

Reply:
I am not fully sure but most probably "errors" is a common case unless explicitly any exception happening.

Reply:
Perhaps in this case it is better let the error occur or give a message in the log, when the key "errors" is not found.

Reply:
@idgserpro agree, current in situation it will let error happen.

Reply:
@nazrulworld no. I meant if the implementation of the result is changed and an error occurs, even if the `errors` key is no longer returned, your code will not notice this change, and will not show the error message. I was telling you to remove the check `"errors" in result`.